storagemarket/types.go

```diff
@@ -102,27 +103,30 @@ type MinerDeal struct {
 
 	DealID       abi.DealID
 	CreationTime cbg.CborTime
+
+	TransferChannelId *datatransfer.ChannelID
```

Comment:
On the provider side, we use a pointer to a channelID, while on the client side, we use a channelID value -- what's the rationale for the difference?

Reply:
I think this is an oversight. Let's use pointer on both sides.